File: 프로그래머스/lv1/42748. K번째수/K번째수.js
function solution(array, commands) {
  var answer = [];
  let arr = [];
  let i = 0;
  let j = 0;
  let k = 0;

  for (let a = 0; a < commands.length; a++) {
    i = commands[a][0] - 1;
    j = commands[a][1] - 1;
    k = commands[a][2] - 1;
    arr = array.slice(i, j + 1).sort((a, b) => a - b);
    answer.push(arr[k]);
    arr = [];
  }

  return answer;
}

File: 프로그래머스/lv1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for(int i = 0; i < commands.length; i++) {
            int[] nums = Arrays.copyOfRange(array, commands[i][0]-1, commands[i][1]);
            Arrays.sort(nums);
            answer[i] = nums[commands[i][2]-1];
        }
        
        return answer;
    }
}